<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wallpaper</class>
 <widget class="QWidget" name="Wallpaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Wallpaper{border-left:1px solid #2d2d2d;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_Wallpaper">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_setting">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_wp">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>600</width>
            <height>410</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>24</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Set wallpaper</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <property name="horizontalSpacing">
                <number>24</number>
               </property>
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <item row="0" column="1">
                <widget class="QWidget" name="widget_lockscreen_preview" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>282</width>
                   <height>112</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>51</number>
                  </property>
                  <property name="topMargin">
                   <number>11</number>
                  </property>
                  <property name="rightMargin">
                   <number>51</number>
                  </property>
                  <property name="bottomMargin">
                   <number>11</number>
                  </property>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QWidget" name="widget_desktop_preview" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>282</width>
                   <height>112</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>51</number>
                  </property>
                  <property name="topMargin">
                   <number>11</number>
                  </property>
                  <property name="rightMargin">
                   <number>51</number>
                  </property>
                  <property name="bottomMargin">
                   <number>11</number>
                  </property>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_desktop_preview">
                 <property name="text">
                  <string>Desktop Wallpaper Preview</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_lockscreen_preview">
                 <property name="text">
                  <string>Lock Screen WallPaper Preview</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Select wallpaper</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="vLayout_chooser">
               <property name="spacing">
                <number>24</number>
               </property>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>140</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_selector">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="label_select_wallpaper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select Wallpaper</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vLayout_image_selector">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
